packages/template-mrt-reference-app/app/ssr.js

```diff
@@ -194,6 +194,17 @@ const cookieTest = async (req, res) => {
     res.json(jsonFromRequest(req))
 }
 
+/**
+ * Express handler that sets a response header and returns a JSON response with
+ * diagnostic values.  Use ?name=test-name&value=test-value to set a response header.
+ */
+const responseHeader = async (req, res) => {
+    if (Object.hasOwn(req.query, 'name') && Object.hasOwn(req.query, 'value')) {
+        res.set(req.query.name, req.query.value)
+    }
+    res.json(jsonFromRequest(req))
+}
```

Comment:
@lizcross – this seems reasonable and useful for a wide range of tests.

I'd not that this won't impact the status code of the response, but thats not really what we're testing.

Reply:
I am going to update my e2e tests in the other PR to not tests redirects directly. The bug was actually about response headers rather than any issue with redirects.